anchore_engine/common/helpers.py

```diff
@@ -366,3 +360,12 @@ def make_eval_record(userId, evalId, policyId, imageDigest, tag, final_action, e
     payload["last_updated"] = payload['created_at']
 
     return payload
+
+def safe_extract_json_value(value):
+    # support the legacy serialized json string
+    if isinstance(value, str):
+        try:
+            value = json.loads(value)
+        except json.decoder.JSONDecodeError:
+            pass
+    return value
```

Comment:
nit: I don't know if there's an established standard with regard to ending files with newlines, but noticed this was a change from the previous state